custom_model_runner/datarobot_drum/drum/root_predictors/prediction_server.py

```diff
@@ -212,6 +209,23 @@ def chat():
 
             return response, response_status
 
+        @model_api.route("/directAccess/<path:path>", methods=["GET", "POST", "PUT"])
+        @model_api.route("/nim/<path:path>", methods=["GET", "POST", "PUT"])
+        def forward_request(path):
+            openai_host = os.environ.get("OPENAI_HOST", "localhost")
+            openai_port = os.environ.get("OPENAI_PORT", "8000")
```

Comment:
This looks like it is generalizing [this](https://github.com/datarobot/datarobot-user-models/blob/fcfd94475e9513b042f26d2d0abb7beda99f01bc/custom_model_runner/datarobot_drum/drum/gpu_predictors/base.py#L66-L68) code.

Can add these strings to [EnvVarNames](https://github.com/datarobot/datarobot-user-models/blob/fcfd94475e9513b042f26d2d0abb7beda99f01bc/custom_model_runner/datarobot_drum/drum/enum.py#L413) and replaces instances with that constant.